src/platform/packages/shared/kbn-apm-synthtrace/src/scenarios/sample_logs.ts

```diff
@@ -55,42 +55,7 @@ const scenario: Scenario<LogDocument> = async (runOptions) => {
           ingest: {
             lifecycle: { inherit: {} },
             processing: {
-              steps: [
-                {
-                  customIdentifier: 'synth-step-0',
-                  action: 'dissect',
-                  where: {
-                    always: {},
-                  },
-                  from: 'attributes.user.name',
-                  pattern: 'user%{attributes.user.id}',
-                  ignore_failure: false,
-                  ignore_missing: false,
-                },
-                {
-                  customIdentifier: 'synth-step-1',
-                  action: 'manual_ingest_pipeline',
-                  where: {
-                    always: {},
-                  },
-                  processors: [
-                    {
-                      convert: {
-                        field: 'attributes.user.id',
-                        type: 'long',
-                        ignore_missing: true,
-                      },
-                    },
-                    {
-                      fail: {
-                        if: 'ctx.attributes?.user?.id != null && ctx.attributes.user.id > 2',
-                        message: 'User is not allowed',
-                      },
-                    },
-                  ],
-                  ignore_failure: false,
-                },
-              ],
+              steps: [],
```

Comment:
Did you remove the `dissect` step on purpose? Or this should only remove the manual_ingest_pipeline one?

Reply:
Done, I removed the field from the mappings in [36166ca](https://github.com/elastic/kibana/pull/234524/commits/36166ca48858b90d547f8023e5bbf77cae3bd4e1).